src/compiler/transformers/ts.ts

```diff
@@ -1304,12 +1307,16 @@ namespace ts {
                     : createNull()
                 : undefined;
 
-            return createDecorateHelper(
+            const helper = createDecorateHelper(
                 decoratorExpressions,
                 prefix,
                 memberName,
-                descriptor
+                descriptor,
+                getUndecoratedRange(member)
             );
+
+            setNodeEmitFlags(helper, NodeEmitFlags.NoComments);

```

Comment:
Should this line be move into "...DecorateHelper"? So we won't have to make sure "setNodeEmitFlags(helper, NodeEmitFlags.NoComments);" is called by the caller every times